src/components/FullTags/FullTagPage.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="canvas-size overflow">
+    <div v-for="(image, index) in imageUrls.length" :key="index">
+      <div class="image-size">
+        <img :src="imageUrls[index].url" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import dataFetch from "../../api/dataFetching";
+
+export default {
+  methods: {
+    loadMoreImages() {
+      for (let i = 2; i < this.nbPage; i++) {
+        dataFetch.getImagesByTag(this.tag, i).then((images) => {
+          this.imageUrls = this.imageUrls.concat(images);
+        });
+      }
+    },
+  },
+  props: {
+    tag: String,
+  },
+  data() {
+    return {
+      imageUrls: [],
+      nbPage: 0,
+    };
+  },
+  created() {
+    // Get the 25 first images
+    dataFetch.getImagesByTag(this.tag, 1).then((images) => {
+      this.imageUrls = images;
+    });
+    // Load all others
+    dataFetch.getNbPagePerTag(this.tag).then((nbPage) => {
```

Comment:
you should probably not load all of them directly in case there would be 1000s of images. We will need some pagination or infinite scroll I guess

Reply:
Yes, I receive yesterday the confirmation that this part should be an infinite scroll. The infinite scroll is now implemented